### DISCUSSION TITLE
Alternative to Silicon Labs cp210x

### DISCUSSION THREAD

**Discussion by xcvator46** [artisan-roaster-scope/artisan#1686] [Q&A]
I've had a power surge that took out my laptop and backup drive, so I'm now in the process of finding all of my old downloads.
I have trouble installing and running Silicon labs drivers and somewhere I was directed to an alternative driver package that was easy to install and run, BUT, I cannot remember the name or place that I downloaded it from.
Has any body have any knowledge of this alternative package to connect my ms6514 to my new laptop please

**Comment by Terracotta-6** [comment]
https://mastech-group.com/as/en/product-list/content/8660B5FB63ee

https://drive.google.com/file/d/1xxtOSYH65pYyCsgVy3exoWY_maN0x1u_/view
